hack/integration_test/README.md

```diff
@@ -0,0 +1,76 @@
+# Integration tests
```

Comment:
Here is a short manual how to use the framework, I suggest reading this first during the review.